ServerNG.md

```diff
@@ -0,0 +1,155 @@
+            <properties path="mgmt-groups.properties" relative-to="infinispan.server.config.dir"/>
+         </authorization>
+      </security-realm>
+      <security-realm name="PublicRealm">
+         <server-identities>
+            <ssl>
+                <keystore path="application.keystore" relative-to="infinispan.server.config.dir" keystore-password="password" alias="server" key-password="password" generate-self-signed-certificate-host="localhost"/>
+            </ssl>
+         </server-identities>
+         <authentication>
+            <properties path="application-users.properties" relative-to="infinispan.server.config.dir"/>
+         </authentication>
+         <authorization>
+            <properties path="application-roles.properties" relative-to="infinispan.server.config.dir"/>
+         </authorization>
+      </security-realm>
+   </security>
+   
+   <!-- JGroups configuration -->
+   <jgroups transport="org.infinispan.remoting.transport.jgroups.JGroupsTransport">
+      <stack-file name="udp" path="jgroups-udp.xml"/>
+      <stack-file name="tcp" path="jgroups-tcp.xml"/>
+   </jgroups>
+   
+   <!-- Cache containers -->
+   <cache-container name="default" ...>
+      <transport .../>
+   </cache-container>
+   
+   <!-- Endpoints -->
+   <endpoints>
+      <hotrod-connector cache-container="default" socket-binding="hotrod" ... />
+      <rest-connector socket-binding="rest" ... />
+      <admin-connector socket-binding="management" security-realm="ManagementRealm" />
+   </endpoints>
+   
+   <socket-bindings>
+      <socket-binding name="hotrod" address="${infinispan.bind.address}" port="11222"/>
+      <socket-binding name="rest" address="${infinispan.bind.address}" port="8080"/>
+      <socket-binding name="management" address="${infinispan.bind.address.management}" port="9990"/>
+      <socket-binding name="cluster" multicast-address="${infinispan.bind.address.multicast}" multicast-port="45700"/>
+   </socket-bindings>
+</infinispan>
+```
+
+# Logging
+
+Logging is handled by JBoss Logging's LogManager. This is configured through a `logging.properties` file in the 
+`server/configuration` directory. The following is an example:
+
+```
+loggers=org.jboss.logmanager
+
+# Root logger
+logger.level=INFO
+logger.handlers=CONSOLE
+
+logger.org.jboss.logmanager.useParentHandlers=true
+logger.org.jboss.logmanager.level=INFO
+
+handler.CONSOLE=org.jboss.logmanager.handlers.ConsoleHandler
+handler.CONSOLE.formatter=PATTERN
+handler.CONSOLE.properties=autoFlush,target
+handler.CONSOLE.autoFlush=true
+handler.CONSOLE.target=SYSTEM_OUT
+
+formatter.PATTERN=org.jboss.logmanager.formatters.PatternFormatter
+formatter.PATTERN.properties=pattern
+formatter.PATTERN.pattern=%d{HH:mm:ss,SSS} %-5p [%c] (%t) %s%E%n
+```
+# Internals
+
+The following is a dump of various internal facts about the server, in no particular order:
+
+* All containers handled by the same server share the same thread pools and transport.
```

Comment:
but you show some `<transport ...>` element in the configuration above...

Reply:
Right, that's a legacy leftover from the current server which allows distinct channels per cache container, but in the name of simplification, that should go.